spec/acceptance/locales_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper_acceptance'
 require 'beaker/i18n_helper'
 
-describe 'mysql localization', if: (fact('osfamily') == 'Debian' || fact('osfamily') == 'RedHat') && (Gem::Version.new(puppet_version) >= Gem::Version.new('4.10.5')) do
+describe 'mysql localization', if: (fact('osfamily') == 'debian' || fact('osfamily') == 'redhat') && (Gem::Version.new(puppet_version) >= Gem::Version.new('4.10.5')) do
```

Comment:
fact('osfamily') will need to be changed to: os[:family]
This is removing facter from the codebase and using 'serverspec' instead, serverspec is faster than facter.